evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -0,0 +1,272 @@
+              (LinearPipelineWithDoubling, {'Select Columns Transformer': {'columns': ['amount']}}),
+              (LinearPipelineWithDoubling, {'Select Columns Transformer': {'columns': ['amount']},
+                                            'DoubleColumns': {'drop_old_columns': False}}),
+              (DagTwoEncoders, {'SelectNumeric': {'columns': ['card_id', 'store_id', 'datetime', 'amount', 'customer_present', 'lat', 'lng']},
+                                'SelectCategorical1': {'columns': ['currency', 'expiration_date', 'provider']},
+                                'SelectCategorical2': {'columns': ['region', 'country']},
+                                'OHE_1': {'features_to_encode': ['currency', 'expiration_date', 'provider']},
+                                'OHE_2': {'features_to_encode': ['region', 'country']}}),
+              (DagReuseFeatures, {'SelectNumeric': {'columns': ['card_id', 'store_id', 'datetime', 'amount', 'customer_present', 'lat', 'lng']},
+                                  'SelectCategorical1': {'columns': ['currency', 'expiration_date', 'provider']},
+                                  'SelectCategorical2': {'columns': ['region', 'country']},
+                                  'OHE_1': {'features_to_encode': ['currency', 'expiration_date', 'provider']},
+                                  'OHE_2': {'features_to_encode': ['region', 'country']},
+                                  'DT': {'encode_as_categories': True}}),
+              (LinearPipelineWithTargetEncoderAndOHE, {'One Hot Encoder': {'features_to_encode': ['currency', 'expiration_date', 'provider']},
+                                                       'Target Encoder': {'cols': ['region', 'country']}}),
+              (LinearPipelineCreateFeatureThenDropIt, {'Select Columns Transformer': {'columns': ['amount']},
+                                                       'DoubleColumns': {'drop_old_columns': False},
+                                                       'Drop Columns Transformer': {'columns': ['amount_doubled']}})
+              ]
+
+
+@pytest.mark.parametrize('pipeline_class, parameters', test_cases)
+@patch('evalml.pipelines.PipelineBase._supports_fast_permutation_importance', new_callable=PropertyMock)
+def test_fast_permutation_importance_matches_sklearn_output(mock_supports_fast_importance, pipeline_class, parameters,
+                                                            has_minimal_dependencies):
+    if has_minimal_dependencies and pipeline_class == LinearPipelineWithTargetEncoderAndOHE:
+        pytest.skip("Skipping test_fast_permutation_importance_matches_sklearn_output for target encoder cause "
+                    "dependency not installed.")
+    X, y = load_fraud(100)
+
+    # Do this to make sure we use the same int as sklearn under the hood
+    random_state = np.random.RandomState(0)
+    random_seed = random_state.randint(np.iinfo(np.int32).max + 1)
```

Comment:
Is this different behavior from what we use in `get_random_seed` / can we just use that? 👀

Reply:
Unfortunately it is! Great suggestion though!